wiki/Tournaments/GTS/CGTS_2019/en.md

```diff
@@ -61,35 +61,205 @@ The Continental Global Taiko Showdown 2019 is run by various community members.
+| L1a¹ | goheegy | ![][flag_GB] | ![][flag_SE] | Nurend | **12:00** |
+| L1b¹ | goheegy | ![][flag_GB] | ![][flag_RU] | pavlukivan | **12:00** |
+| L1c¹ | Nozdormu | ![][flag_RU] | ![][flag_SE] | Nurend | **12:00** |
+| L1d¹ | Nozdormu | ![][flag_RU] | ![][flag_RU] | pavlukivan | **12:00** |
+| L2a² | \[\_LordEnder\_\] | ![][flag_IT] | ![][flag_RU] | Naninub | **13:00** |
+| L2b² | \[\_LordEnder\_\] | ![][flag_IT] | ![][flag_RU] | Akonine | **13:00** |
+| L2c² | Agresywny Arbuz | ![][flag_PL] | ![][flag_RU] | Naninub | **13:00** |
+| L2d² | Agresywny Arbuz | ![][flag_PL] | ![][flag_RU] | Akonine | **13:00** |
+| L3a³ | Maitgon | ![][flag_ES] | ![][flag_DE] | frz | **14:00** |
+| L3b³ | Maitgon | ![][flag_ES] | ![][flag_GB] | Teezel | **14:00** |
+| L3c³ | Nozdormu | ![][flag_RU] | ![][flag_DE] | frz | **14:00** |
+| L3d³ | Nozdormu | ![][flag_RU] | ![][flag_GB] | Teezel | **14:00** |
+| L4a⁴ | Ak1o | ![][flag_DE] | ![][flag_PL] | bernard351 | **15:00** |
+| L4b⁴ | Ak1o | ![][flag_DE] | ![][flag_FR] | maximaxiU | **15:00** |
+| L4c⁴ | StrijkIjzer | ![][flag_NL] | ![][flag_PL] | bernard351 | **15:00** |
+| L4d⁴ | StrijkIjzer | ![][flag_NL] | ![][flag_FR] | maximaxiU | **15:00** |
+| L5a⁵ | Ikkun | ![][flag_IT] | ![][flag_IT] | coed | **16:00** |
+| L5b⁵ | Ikkun | ![][flag_IT] | ![][flag_FI] | duski | **16:00** |
+| L5c⁵ | TwinT | ![][flag_RU] | ![][flag_IT] | coed | **16:00** |
+| L5d⁵ | TwinT | ![][flag_RU] | ![][flag_FI] | duski | **16:00** |
+| L6a⁶ | Genjuro | ![][flag_GR] | ![][flag_FR] | BananaW | **17:00** |
+| L6b⁶ | Genjuro | ![][flag_GR] | ![][flag_MD] | OldFriend | **17:00** |
+| L6c⁶ | cloudfags | ![][flag_RU] | ![][flag_FR] | BananaW | **17:00** |
+| L6d⁶ | cloudfags | ![][flag_RU] | ![][flag_MD] | OldFriend | **17:00** |
+| L7a⁷ | AlvaroYL | ![][flag_ES] | ![][flag_FI] | vodnanen | **18:00** |
+| L7b⁷ | AlvaroYL | ![][flag_ES] | ![][flag_FR] | -Akitai- | **18:00** |
+| L7c⁷ | Gintoki8 | ![][flag_FR] | ![][flag_FI] | vodnanen | **18:00** |
+| L7d⁷ | Gintoki8 | ![][flag_FR] | ![][flag_FR] | -Akitai- | **18:00** |
+| L8a⁸ | D3kuu | ![][flag_IT] | ![][flag_PL] | Acrith | **19:00** |
+| L8b⁸ | D3kuu | ![][flag_IT] | ![][flag_RU] | Den4ik228 | **19:00** |
+| L8c⁸ | Dayzeek | ![][flag_RU] | ![][flag_PL] | Acrith | **19:00** |
+| L8d⁸ | Dayzeek | ![][flag_RU] | ![][flag_RU] | Den4ik228 | **19:00** |
+
+¹ Encounter dependent on winner of match W1 and match W2
+
+² Encounter dependent on winner of match W3 and match W4
+
+³ Encounter dependent on winner of match W5 and match W6
+
+⁴ Encounter dependent on winner of match W7 and match W8
+
+⁵ Encounter dependent on winner of match W9 and match W10
+
+⁶ Encounter dependent on winner of match W11 and match W12
+
+⁷ Encounter dependent on winner of match W13 and match W14
+
+⁸ Encounter dependent on winner of match W15 and match W16
+
+---
 
 ## Mappools
 
-*Each mappools will be announced every 7 days (usually Sunday) before the next stage's match timeline.*
+### Round of 32
```

Comment:
```suggestion
### Regional phase

#### Round of 32
```
kinda sucks that this will make Ro32 unable to be linked directly but idk how else you'd format this.